### ISSUE TITLE
New feature: get the dependent gems of a particular gem

### ISSUE THREAD

**Issue by v0dro** [zverok/any_good#6]
Getting the names of the gems that use a given gem and the version number of the gem that the dependents use is useful information for gem writers. It would also be great to provide details about the proportion of the downloads of the dependent with a given gem to see which dependent gem is most important by number of downloads.

**Comment by zverok** [comment]
Yes, thought about that too, it would allow to easily catch outliers ("yes, this gem seems to be downloaded a lot, but that's just because `aws` gem uses it in one of tests").

The problem is RubyGems doesn't have any bulk API, so for something like `activesupport` it would look like "Ugh, wait please we'll just make 9k API calls to analyze the dependencies". So, probably this option should work... IDK, "sometimes"? Like when there are not a lot of dependencies or something. But in this case, you already have pretty good guess of "if this any good" :) I want to have it simple.

**Comment by v0dro** [comment]
We can query every gem dependency on rubygems in a multi-threaded manner and gather the data in parallel. Then analyse it and display. This can be an optional feature since sending so many queries to rubygems is obviously time consuming.

**Comment by arbox** [comment]
We could talk to the Rubygems people and try to introduced convenient API for this tasks. RubyToolbox and Libraries.io would benefit through it as well.

**Comment by v0dro** [comment]
@arbox do you know someone who can help us out? I can write them a mail.

**Comment by arbox** [comment]
It could be e.g. @indirect
